bazel/fuzzing_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Dummy shell implementation for fuzzing test.
+
+if [ $# -gt 0 ]; then
+  "$@"
+fi
```

Comment:
Somebody reading this would have a hard time understanding what's happening. Consider parsing a bit the command line, e.g.:

```
readonly LAUNCHER_PATH="$1"; shift
exec "$LAUNCHER_PATH" --regression "$@"
```